supporting-blog-content/agentic-rag/agent_rag_news_assistant.ipynb

```diff
@@ -288,17 +271,16 @@
    "id": "d78b4e82ff95374d",
    "metadata": {
     "ExecuteTime": {
-     "end_time": "2025-10-29T14:27:18.437609Z",
-     "start_time": "2025-10-29T14:27:10.616426Z"
+     "end_time": "2025-11-06T15:07:58.647936Z",
+     "start_time": "2025-11-06T15:07:53.046956Z"
     }
    },
    "source": [
-    "index_name=\"news_docs\"\n",
+    "index_name = \"news_docs\"\n",
     "elastic_vectorstore = ElasticsearchStore.from_documents(\n",
     "    docs,\n",
-    "    es_url=os.environ[\"ES_URL\"],\n",
-    "    es_user=os.environ[\"ES_USERNAME\"],\n",
-    "    es_password=os.environ[\"ES_PASSWORD\"],\n",
+    "    es_url=ES_ENDPOINT,\n",
+    "    es_api_key=ES_API_KEY,\n",
```

Comment:
@sodhikirti07 , I found my problem: it was truly a connection timeout error on Elastic's part. This may happen to others, so let's extend the timeout for elasticsearch to consume the documents. I added this to the arguments for this `ElasticsearchStore.from_documents` and everything ran perfectly after that:

```python
    es_params={
        "request_timeout": 60,  # Timeout in seconds
        "max_retries": 3,       # Number of retries
        "retry_on_timeout": True  # Retry on timeout
    }
```
Add this below the es_api_key call and everything should be good.

Reply:
Thanks for checking. Added the params to `ElasticsearchStore.from_documents`. I'm not sure why the pre-commit is failing as I already run `precommit install`.